src/components/PostComment.jsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+import { postComment } from "../utils/api";
+import moment from "moment";
+
+export const PostComment = ({ review_id }) => {
+  const [body, setBody] = useState("");
+  let username = "jessjelly";
+
+  const handleSubmit = (e) => {
+    if (body.length < 10 || username === "") {
+      e.preventDefault();
+      alert("Comments must be 10 characters or more");
+    } else {
+        document.getElementById("comment-body").disabled = true;
```

Comment:
you shouldn't be doing any DOM malipulation in React!

You should be using state & JS logic to control attributes like `disabled`

This needs to be refactored